Sources/Core/Other/PasteboardMonitor.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  PasteboardMonitor.swift
+//  XestiMonitors
+//
+//  Created by Paul Nyondo on 2018-03-15.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+#if os(iOS)
+
+    import  UIKit
+
+    ///
+    /// A `PasteboardMonitor` instance monitors a pasteboard for changes to
+    /// its state.
+    ///
+    public class PasteboardMonitor: BaseNotificationMonitor {
+        ///
+        /// Encapsulates changes to the contents of the pasteboard.
+        ///
+        public enum Event {
+            ///
+            ///  Contents of the pasteboard object have changed.
+            ///
+            case didChange(Info)
+
+            ///
+            ///  Contents of the pasteboard object have been removed.
+            ///
+            case didRemove(Info)
```

Comment:
Let’s rename this to `removed` to better match the notification name.

Also, the notification is sent when the _pasteboard_ itself is removed, _not_ the contents of the pasteboard. Therefore, the associated value for this event should be `UIPasteboard`. Thus:
```
case removed(UIPasteboard)
```

Reply:
How should we now handle the `Info` struct so that we can access the `userInfo` information